cfpb/capital-framework
# HOLD: Remove the deprecated font-icon system

The font-icons were deprecated when the SVG icon system was released.
Now that most projects have been updated it's time to remove the old
icon font and related styles and documentation.

## Removals

-  Removed the deprecated font-icon system from cf-icons

## Testing

1. Follow the [testing locally instructions](https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#testing-components-locally)
1. Run `gulp build` && `npm start`
1. Navigate to `http://localhost:3000/components/cf-icons`
1. Confirm the icon font documentation is gone.

## Notes

- Should be released along with #787 

## Todos

- Double check cfgov-refresh has not had font icons added back into the project since it was updated for CFv5

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
